prestashop6.go

```diff
@@ -0,0 +1,62 @@
+package gocommerce
+
+import (
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+type Prestashop6 struct {
+	basePlatform
+}
+
+var (
+	p6VerRgx    = regexp.MustCompile(`define\('VERSION',\s?'([^']+)'\);`)
```

Comment:
I guess we can comment it for now

```suggestion
	//p6VerRgx    = regexp.MustCompile(`define\('VERSION',\s?'([^']+)'\);`)
```

Reply:
Oops I copied this from a different platform file with the intent to implement it... But do we really need the specific 1.6 version? Not worth..